CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# How to contribute to U.S. Digital Response projects
+
+Everyone is welcome to contribute, and we value everybody's contribution. Code is not the only
+way to help the community. Answering questions, helping others, reaching out, and improving
+documentation are all immensely valuable to the community.
+
+
+## You can contribute in so many ways!
+
+There are several ways you can contribute to this repository:
+
+- Enhancing and/or fixing outstanding issues with the existing code
+- Contributing useful improvements, clarifications, and updates to existing documentation
+- Submitting [issues](https://github.com/usdigitalresponse/grants-ingest/issues/new/choose)
+related to bugs or desired new features
+- Reporting security vulnerabilities
+
+_All are equally valuable to the community._
+
+We also onboard new volunteers to USDR projects in general at https://www.usdigitalresponse.org/volunteer.
+Please sign up if you’d like to help on other projects.
+
+
+## Submitting a new issue or feature request
+
+Do your best to follow these guidelines when submitting an issue or a feature request. It will
+make it easier for us to come back to you quickly and with good feedback.
```

Comment:
```suggestion
Do your best to follow these guidelines when submitting an issue or a feature request. 
It will make it easier for us to come back to you quickly and with good feedback.
```